weaver/config.py

```diff
@@ -16,10 +16,16 @@
 WEAVER_CONFIGURATION_DEFAULT = "DEFAULT"
 WEAVER_CONFIGURATION_ADES = "ADES"
 WEAVER_CONFIGURATION_EMS = "EMS"
+WEAVER_CONFIGURATION_HYBRID = "HYBRID"
```

Comment:
Maybe add an explanation in the Configuration Settings docs to know a little bit more about the hybrid mode.